src/org/zaproxy/zap/extension/websocket/ui/WebSocketUiHelper.java

```diff
@@ -497,7 +528,94 @@ public JCheckBox getRegexCheckbox() {
 		return regexCheckbox;
 
 	}
+
+	//************************** MANUAL SEND DIALOG ***************************
+	
+	public JLabel getConnectingLabel(ImageIcon icon){
```

Comment:
The icon is not being set.

I don't have a preference for the icon location, if you want to keep the previous location you can do so by changing the horizontal text position (to leading).